samples/grids/hierarchical-grid/toolbar-sample-4/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Sample | Ignite UI | Web Components | infragistics</title>
+  <meta charset="UTF-8" />
+
+  <link rel="shortcut icon" href="https://static.infragistics.com/xplatform/images/browsers/wc.png">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Kanit&display=swap" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium Web" />
+  <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v8.css" />
+  <link rel="stylesheet" href="/src/index.css" type="text/css" />
+
+</head>
+
+<body>
+  <div id="root">
+    <div class="container sample">
+      <div class="container fill">
+
+        <igc-hierarchical-grid auto-generate="false" name="hGrid1" id="hGrid1">
+          <igc-grid-toolbar>
+            <igc-grid-toolbar-title>Singers</igc-grid-toolbar-title>
+            <igc-button id="clearSort">
+              <igc-icon name="clear" collection="material"></igc-icon>Clear Sort
+            </igc-button>
```

Comment:
Missing toolbar actions in comparison to grid samples, but probably ok.

Reply:
I added column hiding and toolbar for the child grid to align with the [Angular sample](https://www.infragistics.com/products/ignite-ui-angular/angular/components/hierarchicalgrid/toolbar#custom-content).